lib/public_suffix/rules_parser.ex

```diff
@@ -58,7 +58,7 @@ defmodule PublicSuffix.RulesParser do
   @doc false
   def punycode_domain(rule) do
     rule
-    |> :xmerl_ucs.from_utf8
+    |> :unicode.characters_to_list
```

Comment:
I copied this from the [idna README quick start](https://github.com/tim/erlang-idna#quick-start).  I'm not familiar with either `xmerl_ucs` or `unicode` and I'm not sure I feel comfortable merging this change w/o understanding what, if anything, this will affect.  Are you sure they are equivalent?  If there's an advantage to using `unicode` over `xmerl_ucs`, does it make sense to ask the author of erlang-idna to update their quick start to recommend using `unicode` instead?

Your commit mentions that xmerl is an extra package whereas unicode is built in....but I didn't have to install a hex package to use xmerl and it's part of the [erlang.org docs](http://erlang.org/doc/man/xmerl.html) which makes me think it's built in to erlang's standard library.  Can you expand on what you mean?

Bear in mind that I have little experience in the Erlang ecosystem, so I'm a bit out of my element in being able to judge if this is a good change or not.  Anything you can do to help me understand your rationale behind it and what the repercussions for end users are is much appreciated :).